packages/fxa-settings/src/components/Settings/ProductPromo/index.tsx

```diff
@@ -24,24 +24,51 @@ export interface ProductPromoProps {
 export const ProductPromo = ({
   type = ProductPromoType.Sidebar,
 }: ProductPromoProps) => {
-  const { attachedClients } = useAccount();
+  const { attachedClients, subscriptions } = useAccount();
 
   const hasMonitor = attachedClients.some(
     ({ name }) => name === MozServices.Monitor
   );
-  // const hasMonitorPlus = subscriptions.some(
-  //   ({ productName }) => productName === MozServices.MonitorPlus
-  // );
+  const hasMonitorPlus = subscriptions.some(
+    ({ productName }) => productName === MozServices.MonitorPlus
+  );
 
-  if (hasMonitor) {
+  if (hasMonitor && hasMonitorPlus) {
```

Comment:
One thing I wasn't sure about here - while this seems right since if users have all products we don't want to display the promo, is there a case where the user might not be logged into Monitor but they have a Plus subscription? I'm guessing we wouldn't want to show the promo in that case either but not sure if it's worth checking for this case until we offer this for more than just Monitor.

Reply:
I can confirm, but I don't think it should be possible to have the Plus subscription without Monitor.

Reply:
I was thinking if someone had signed out of Monitor, but they still had the subscription. I guess it's not a big deal if they get the free version advertised to them in this case?